src/client/components/checkout/Footer.stories.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+import { MemoryRouter, Route, RouterProps } from 'react-router-dom'
+import { LocaleLabel } from 'l10n/locales'
+import {
+  mockedQuoteCartId,
+  mockedPriceQueryResponse,
+} from 'utils/testData/priceQueryMock'
+import { TextKeyProvider, TranslationsLocale } from 'utils/textKeys'
+import { PriceDocument, Locale } from 'data/graphql'
+import { localePathPattern } from 'l10n/localePathPattern'
+import { Footer, Props as FooterProps } from './Footer'
+
+type StoryProps = {
+  localePath: LocaleLabel
+} & FooterProps
+
+const storyMeta: Meta<StoryProps> = {
+  title: 'Components/Checkout/Footer/Footer',
```

Comment:
Looking at the structure of components, shouldn't we keep all components that are specific to Checkout under something like `pages/Checkout/..`? To me it makes more sense, since `components/` usually consists of generic components that are not tied to specific views or pages.

Reply:
Hmm, yeah maybe. These components that I put under "Components" are there since that's where they live in the folder structure. And the reason for that is that we have two checkout pages that share some components (like this one for example). So it can't really be placed in one page folder 🤔 

But maybe our Storybook folder structure doesn't have to mirror the actual folder structure? Or we could have a `Checkout` folder under `pages` with a `Details`, `Payment` and `shared` folder? What do you think?

Reply:
OK so I fixed it in Storybook and put the `Footer` directly below `Checkout`. After some consideration I think the repo folder structure should also be changes to what I suggested above. But I will do that in another PR.